opm/parser/eclipse/EclipseState/Grid/MinpvMode.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+  Copyright 2015 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _MINPVMODE_
+#define _MINPVMODE_
+
+#include <string>
+
+namespace Opm {
+
```

Comment:
1. Please put the enum in a separate namespace - see e.g. the `ScheduleEnums.hpp` file.
2. Unless I am completely mistaken these values should be used one-at-a-time; i.e. not as bitmasks. The use of numerical values 1,2,4,  hints at bitmask usage and can be misleading, please just use 1,2,3 or 0,1,2